services/api/src/api/routes/endpoint.py

```diff
@@ -27,8 +29,20 @@
 )
 
 
-def create_processing_step_endpoint(
-    processing_step: ProcessingStep,
+class EndpointsDefinition:
+    """Definition of supported endpoints and its relation with processing steps."""
+
+    definition: Mapping[str, List[ProcessingStep]]
```

Comment:
```suggestion
    processing_steps_by_endpoint: Mapping[str, List[ProcessingStep]]
```

or something similar, more descriptive?